alacritty_terminal/src/tty/windows/winpty.rs

```diff
@@ -32,8 +31,9 @@ use crate::tty::windows::Pty;
 
 pub use winpty::Winpty as Agent;
 
-/// How long the winpty agent should wait for any RPC request
-/// This is a placeholder value until we see how often long responses happen
+/// How long the winpty agent should wait for any RPC request.
+///
+/// This is a placeholder value until we see how often long responses happen.
```

Comment:
If you don't like the comment changes let me know.

Reply:
I'm not sure if this second comment is still needed? It seems like it's working just fine.

Reply:
cc @davidhewitt

Reply:
We might be able to remove the `set_agent_timeout` call completely; the default timeout is slightly longer but probably also fine:

https://github.com/rprichard/winpty/blob/7e59fe2d09adf0fa2aa606492e7ca98efbc5184e/src/libwinpty/WinptyInternal.h#L49

Reply:
Removing stuff sounds good to me.